packages/test/test-version-utils/src/compatUtils.ts

```diff
@@ -42,6 +49,79 @@ import {
  */
 export const TestDataObjectType = "@fluid-example/test-dataStore";
 
+/**
+ * This function modifies container runtime options according to a version of runtime used.
+ * If a version of runtime does not support some options, they are removed.
+ * If a version runtime supports some options, such options are enabled to increase a chance of
+ * hitting feature set controlled by such options, and thus increase chances of finding product bugs.
+ *
+ * @param version - a version of container runtime to be used in test
+ * @param optionsArg - input runtime options (optional)
+ * @returns - runtime options that should be used with a given version of container runtime
+ * @internal
+ */
+function filterRuntimeOptionsForVersion(
+	version: string,
+	optionsArg: IContainerRuntimeOptions = {
+		summaryOptions: {
+			summaryConfigOverrides: {
+				...DefaultSummaryConfiguration,
+				...{
+					initialSummarizerDelayMs: 0,
+				},
+			},
+		},
+	},
+) {
+	let options = { ...optionsArg };
+
+	// No test fails with this option, it allows us to validate properly expectations and
+	// implementation of services
+	options.loadSequenceNumberVerification = "close";
+
+	const compressorDisabled: ICompressionRuntimeOptions = {
+		minimumBatchSizeInBytes: Number.POSITIVE_INFINITY,
+		compressionAlgorithm: CompressionAlgorithms.lz4,
+	};
+
+	// These is the "maximum" config.
+	const {
+		compressionOptions = {
+			minimumBatchSizeInBytes: 200,
+			compressionAlgorithm: CompressionAlgorithms.lz4,
+		},
+		chunkSizeInBytes = 200,
+		enableRuntimeIdCompressor = "on",
+		enableGroupedBatching = true,
+	} = options;
+
+	if (version === "1.3.7") {
```

Comment:
What is special about the `1.3.7` version that we have to deal with it separately?

Reply:
That's LTS version that we test, it does not support any latest features.
But maybe your question is - why so precise and what happens when we have a patch? I'll change it to be more inclusive and capture any 1.x.y